packages/legacy/core/App/localization/pt-br/index.ts

```diff
@@ -254,7 +254,13 @@ const translation = {
     "EnableBiometrics": "Você deve habilitar a biometria para poder carregar a carteira.",
     "BiometricsNotProvided": "Biometria não informada, você deve usar o PIN para carregar a carteira.",
     "BiometricsError": "Biometrics were not successful.",
-    "BiometricsErrorEnterPIN": "Please enter your wallet PIN."
+    "BiometricsErrorEnterPIN": "Please enter your wallet PIN.",
```

Comment:
Are you able to add `(PT-BR)` to then end of these so if they're seen in the app we know where they come from?